src/main/features/linux/mprisService.js

```diff
@@ -0,0 +1,84 @@
+import { app } from 'electron';
+import mpris from 'mpris-service';
+import _ from 'underscore';
+
+function mprisService() {
+  const mainWindow = WindowManager.getAll('main')[0];
+  let _songInfo = {};
+
+  const player = mpris({
+    name: 'google-play-music-desktop-player',
+    identity: 'Google Play Music Desktop Player',
+    canRaise: true,
+    supportedInterfaces: ['player'],
+    desktopEntry: 'google-play-music-desktop-player',
+  });
+  player.playbackStatus = 'Stopped';
+
+  player.on('raise', () => {
+    mainWindow.setSkipTaskbar(false);
+    mainWindow.show();
+  });
+
+  player.on('quit', () => {
+    app.quit();
```

Comment:
Something to think about here, should this act as a quit, even if we have minimize to task bar turned on?

(I'm not sure how this quit command would be received)